doc/modules/feature_extraction.rst

```diff
@@ -101,6 +101,88 @@ memory the ``DictVectorizer`` class uses a ``scipy.sparse`` matrix by
 default instead of a ``numpy.ndarray``.
 
 
+.. _column_transformer:
+
+ColumnTransformer for heterogeneous data
+========================================
+
+.. warning::
+
+    The :class:`experimental.ColumnTransformer <sklearn.experimental.ColumnTransformer>`
+    class is experimental and the API is subject to change.
+
+Many datasets contain features of different types, say text, floats, and dates,
+where each type of feature requires separate preprocessing or feature
+extraction steps.  Often it is easiest to preprocess data before applying
+scikit-learn methods, for example using `pandas <http://pandas.pydata.org/>`__.
+Processing your data before passing it to scikit-learn might be problematic for
+one of the following reasons:
+1. Incorporating statistics from test data into the preprocessors makes
+   cross-validation scores unreliable (known as *data leakage*).
+2. You may want to include the parameters of the preprocessors in a
+   :ref:`parameter search <grid_search>`.
+:class:`~sklearn.experimental.ColumnTransformer` helps performing different
+transformations for different columns of the data, within a
+:class:`~sklearn.pipeline.Pipeline` that is safe from data leakage and that can
+be parametrised. ColumnTransformer works on arrays, sparse matrices, and pandas
```

Comment:
I would use ``parametrized``, I think we generally do US English, right?  also :class:`ColumnTransformer`.